cartographi/config_validation/mesh_schema.py

```diff
@@ -40,27 +40,19 @@
 
 mesh_schema = {
     "type": "object",
-    "required": ["Mesh_info"],
-    "additionalProperties": False,
+    "required": ["Region", "Data_sources", "splitting"],
```

Comment:
Since we're breaking everything and separating out the repos anyway it seems like a good time to standardise the case for naming parameters in our configs. I seem to remember in the past we agreed to match this to PEP8 naming conventions for variables (i.e. snake case). So, for example, this line would become: ```"required": ["region", "data_sources", "splitting"]``` What do you think @SamuelHall700 ?

Reply:
I agree; the configs for a mixture of all sorts of stylings right now. It would probably be best to stick to pep8 everywhere